bundles/core/org.eclipse.smarthome.core.test/src/test/java/org/eclipse/smarthome/core/library/types/DateTimeTypeTest.java

```diff
@@ -166,13 +168,35 @@ public void serializationTest() {
     }
 
     @Test
+    public void serializationTestZoned() {
+        ZonedDateTime zoned = ZonedDateTime.now();
+        DateTimeType dt = new DateTimeType(zoned);
+        DateTimeType sdt = new DateTimeType(dt.toString());
```

Comment:
According to the `Type` documentation, this formally should be `DateTimeType.valueOf(dt.toFullString())`. Although you are right that in this case the result is the same because of the delegation. However, if you go strictly by the contract, this does not necessarily have to be like that and the symmetrie is only required for `toFullString()`.

Reply:
@doandzhi do you plan to change this to `toFullString()` at least?